tests/dailytest/case.go

```diff
@@ -106,18 +106,34 @@ var case2Clean = []string{`
 }
 
 var case3 = []string{`
-CREATE TABLE a(id INT PRIMARY KEY, a1 INT);
+CREATE TABLE binlog_case3(id INT PRIMARY KEY, a1 INT);
 `,
 	`
-INSERT INTO a(id, a1) VALUES(1,1),(2,1);
+INSERT INTO binlog_case3(id, a1) VALUES(1,1),(2,1);
 `,
 	`
-ALTER TABLE a ADD UNIQUE INDEX aidx(a1);
+ALTER TABLE binlog_case3 ADD UNIQUE INDEX aidx(a1);
 `,
 }
 
 var case3Clean = []string{
-	`DROP TABLE a`,
+	`DROP TABLE binlog_case3`,
+}
+
```

Comment:
add some comment context like the origin "# Verify fix for TOOL-1346"

Reply:
> add some comment context like the origin "# Verify fix for TOOL-1346"

Already added